### ISSUE TITLE
Make mapml-extension UI localize with the browser

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/mapml-extension#18]
Could we integrate the UI locale strings into the same or different json files, perhaps?

**Comment by ahmadayubi** [comment]
Yeah definitely, it can be integrated into the same .json file translations. I held off on it because the extension UI is currently more for debugging than production so I didn't want to have translations for something that will likely change. 

So before this step it may be useful in deciding what the UI should look like, there is no standard design, it's completely up to extension developers and what they feel works best.
